source/transfer_list.rst

```diff
@@ -657,5 +658,86 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+.. _arm_entries:
+
+Arm specific entries
+^^^^^^^^^^^^^^^^^^^^
+
+The following entry types are defined for Arm projects,
+including TF-A, OP-TEE and FF-A:
```

Comment:
Suggest "The following entry types are defined for TF-A related projects, including OP-TEE". I don't think FF-A should be mentioned here.

Reply:
SPMC manifest is a term from FF-A spec.

Reply:
Fixed.